specification/advisor/resource-manager/Microsoft.Advisor/stable/2020-01-01/advisor.json

```diff
@@ -142,11 +144,26 @@
           },
           {
             "$ref": "#/parameters/subscriptionIdParameter"
+          },
+          {
+            "name": "configurationName",
+            "in": "path",
+            "description": "Advisor configuration name. Value must be 'default'",
+            "required": true,
+            "type": "string",
+            "enum": ["default"],
+            "x-ms-enum": {
+              "name": "ConfigurationName",
+              "modelAsString": true
+            }
           }
         ],
         "responses": {
-          "204": {
-            "description": "No Content. Successfully created/overwrote configuration."
+          "200": {
+            "description": "Successfully created/overwrote configuration.",
+            "schema": {
+              "$ref": "#/definitions/ConfigData"
+            }
           },
           "400": {
```

Comment:
The 4xx/5xx error responses should be removed. The "default" response should handle all error responses with a schema following the common error contract.